src/extension.ts

```diff
@@ -94,10 +94,16 @@ export function activate(context: ExtensionContext) {
         const diagnosticsForFiles: Diagnostic[] =
           convertLinesToDiagnostics(coverage.lines.details);
 
+        let fileName = coverage.file;
+        const is_absolue_path = fileName.length > 2 && (fileName[0] === "/" || fileName[1] === ":");

```

Comment:
So if anyone knows a nicer way to do this I'm all ears!

Reply:
We could probably use https://nodejs.org/api/path.html#path_path_isabsolute_path here

Reply:
Please use camelCase for variable names.